app/src/main/res/values/strings.xml

```diff
@@ -571,6 +571,32 @@
     <string name="update_profile_alert">Please complete your profile to enjoy full features</string>
     <string name="other_notes">Other Notes</string>
     <string name="archive">Archive</string>
+    <string name="Q1">Question : 1/</string>
+    <string name="no_questions">No questions available</string>
+    <string name="Q">"Question : "</string>
+    <string name="finish">Finish</string>
+    <string name="submit">Submit</string>
+    <string name="our_courses">Our Courses</string>
+    <string name="file">"File : "</string>
+    <string name="recording_audio">Recording audio......</string>
+    <string name="show_replies">"Show replies "</string>
+    <string name="selected">"Selected : "</string>
+    <string name="last_sync"><![CDATA["<< Last sync with server: "]]></string>
+    <string name="last_syncs">LastSync</string>
+    <string name="login_user">loginUserName</string>
+    <string name="login_password">loginUserPassword</string>
+    <string name="no_team_available">No team / enterprise available</string>
+    <string name="last_sync_date">"Last Sync Date: "</string>
+    <string name="no_assignee">No assignee</string>
+    <string name="note_">notes</string>
+    <string name="diagno">diagnosis</string>
```

Comment:
Like here

Reply:
<string name="observation">Observations and Notes</string><resources translatable="false"></resources>
Would this work? Or did I misunderstand the formatting?

Reply:
They are to be in this formate under only **app/src/main/res/values/strings.xml** 
<string name="observation" translatable="false">Observations and Notes</string>